apps/explorer/src/ui/SenderRecipientAddress.tsx

```diff
@@ -20,8 +20,10 @@ export function SenderRecipientAddress({
             <div className="w-4 mt-1">
                 {isSender ? <Sender /> : <Recipient />}
             </div>
+
             <Link
                 variant="mono"
+                size="md"
```

Comment:
Should probably be the default right?

Reply:
No, right now, size just overwrites the default font size for either `mono` or `text.`  I will revisit it later.